### PR TITLE
feature[RENA-9200]: Add and update color in base-theme.ts

### PR DESCRIPTION
**Pull request by JesusMacedo-Harleigh**
Add and update color(purple) in base-theme.ts

https://moveinc.atlassian.net/jira/software/c/projects/RENA/boards/929?assignee=712020%3Af63f42af-2dfc-4ef3-9675-ca906ee342f1&selectedIssue=RENA-9200

### CODE REVIEW

**Review by tpstar — COMMENTED** [review]
@JesusMacedo-Harleigh Do we want to do a version control? e.g. Updating the core package.json version? So, we can test updating the colors in the rentalutions repo?
The current "@rent_avail/core": "0.0.6", is using 0.0.6 version. And the current elements core version is 0.0.8. So, we might not need to upgrade the core version since 0.08 is not applied to the rentalutions yet?
If you just merge your changes to release branch, that change will be in 0.0.8 version? And since the rentalutions is using 0.06 core, your change won't be reflected? 
I don't exactly sure how the version control works. Can you check how your changes will be reflected on rentalutions side?

File: packages/core/src/base-theme.ts [hunk 1]
```diff
@@ -33,9 +33,12 @@ const baseColors = {
 
   red_300: "#F6A5B4",
   red_500: "#EC4261",
+  red_100: "#FDECF0",
```
Comment (tpstar) [inline packages/core/src/base-theme.ts:36]: nit pick I would put red_100 above red_300

File: packages/core/src/base-theme.ts [hunk 2]
```diff
@@ -33,9 +33,12 @@ const baseColors = {
...
 
-  purple_300: "#B7B5EE",
-  purple_500: "#6965D5",
+  purple_300: "#BEBDF0",
+  purple_500: "#5D59D9",
+  purple_700: "#262296",
+  purple_100: "#EFEEFB"
```
Comment (tpstar) [inline packages/core/src/base-theme.ts:41]: nit pick place purple_100 above purple_300

**Review by tpstar — COMMENTED** [review]
@JesusMacedo-Harleigh Nice job. LGTM
a nit pick before I approve, can you place _100 above _300?

**Review by tpstar — APPROVED** [review]
@JesusMacedo-Harleigh Thanks for addressing my comments. LGTM

**Comment by JesusMacedo-Harleigh** [comment]
> @JesusMacedo-Harleigh Thanks for addressing my comments. LGTM

Thanks to you @tpstar  for taking the time to make Code review :D ❤️

**Review by armando1339-realtor — APPROVED** [review]
LGTM!

**Review by sarahkazi0627 — APPROVED** [review]

**Pull request merged**